sponsor.html

```diff
@@ -369,22 +488,22 @@ <h2> Sponsorship Prospectus </h2>
       <div class="social">
         <ul class="list-inline text-center">
           <li class="list-inline-item">
-            <a class="social-link" href="#"></a>
+            <a class="social-link" target="_blank" href="https://www.youtube.com/channel/UC-Wo4UqhGUKQEp4o-51VMIw/"><img src="./img/w_youtube.svg"></a>
           </li>
           <li class="list-inline-item">
-            <a class="social-link" href="#"></a>
+            <a class="social-link" target="_blank" href="https://twitter.com/gotechnica?lang=en"><img src="./img/w_twitter.svg"></a>
           </li>
           <li class="list-inline-item">
-            <a class="social-link" href="#"></a>
+            <a class="social-link" target="_blank" href="https://www.facebook.com/gotechnica/"><img src="./img/w_facebook.svg"></a>
           </li>
           <li class="list-inline-item">
-            <a class="social-link" href="#"></a>
+            <a class="social-link" target="_blank" href="https://www.instagram.com/gotechnica/"><img src="./img/w_instagram.svg"></a>
           </li>
           <li class="list-inline-item">
-            <a class="social-link" href="#"></a>
+            <a class="social-link" target="_blank" href="https://medium.com/gotechnica"><img src="./img/w_medium.svg"></a>
           </li>
           <li class="list-inline-item">
```

Comment:
![Screen Shot 2020-04-02 at 10 34 51 PM](https://user-images.githubusercontent.com/30019505/78318306-674aee80-7532-11ea-932a-3e3227b0a49e.png)

Hmmm, looks like the images aren't rendering for me. Am I doing something wrong?

Reply:
no, i added the images in the other branch (content) so i wasn't sure if adding the same images in here would cause a merge conflict once both branches are merged